src/Logging/composer.json

```diff
@@ -4,11 +4,9 @@
     "license": "Apache-2.0",
     "minimum-stability": "stable",
     "require": {
-        "google/cloud-core": "^1.0"
-    },
-    "suggest": {
-        "google/gax": "Required to support gRPC",
-        "google/proto-client": "Required to support gRPC"
+        "google/cloud-core": "^1.0",
+        "google/proto-client": "^0.23.0",
+        "google/gax": "^0.23.0"
```

Comment:
supernit, but why are we including the `.0` bug version in the dependencies? Leaving it out would have the same effect.

Reply:
Dropping `.0` sounds good to me as well

Reply:
Done